jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcCrudRepository.java

```diff
@@ -97,13 +95,14 @@ public int saveBatch(List<T> items) {
      * It uses an update statement, so the item must be already present in the database.
      */
     public T update(T current) {
+        String uid = current instanceof HasUID hasUID ? hasUID.uid() : null;
```

Comment:
In fact here, update cannot be done if the entity cannot provide a UID.
So you must throw defensively if it's not the case.